app/AvailablePlugin/ElectorDataFilter/View/ElectorDataFilterPrecedences/add.ctp

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
Are these view files actually files or symlinks? (They should just be symlinks to the Standard view directory.)